src/de/jost_net/JVerein/gui/view/MailVorlageDetailView.java

```diff
@@ -43,6 +53,8 @@ public void bind() throws Exception
     ButtonArea buttons = new ButtonArea();
     buttons.addButton("Hilfe", new DokumentationAction(),
         DokumentationUtil.MAILVORLAGE, false, "question-circle.png");
+    buttons.addButton("Variablen anzeigen (F6)",
```

Comment:
Könntest du die F6 entfernen. Ich habe das in#450 auch bei den Lesefeldern entfernt. Damit haben wir es einheitlich in allen Views.

Wenn wir das einführen, sollten wir es überall machen.

Reply:
Klar, habe ich einfach nur übernommen.